src/main/java/com/conveyal/r5/streets/LinkedPointSet.java

```diff
@@ -185,10 +187,17 @@ public LinkedPointSet(LinkedPointSet sourceLinkage, WebMercatorGridPointSet subG
             for (int x = 0; x < subGrid.width; x++, pixel++) {
                 int sourceColumn = subGrid.west + x - superGrid.west;
                 int sourceRow = subGrid.north + y - superGrid.north;
-                int sourcePixel = sourceRow * superGrid.width + sourceColumn;
-                edges[pixel] = sourceLinkage.edges[sourcePixel];
-                distances0_mm[pixel] = sourceLinkage.distances0_mm[sourcePixel];
-                distances1_mm[pixel] = sourceLinkage.distances1_mm[sourcePixel];
+                if (sourceColumn < 0 || sourceColumn >= superGrid.width || sourceRow < 0 || sourceRow >= superGrid.height) { //point is outside super-grid
+                    edges[pixel] = -1; //set the edge value to -1 to indicate no linkage.
+                    distances0_mm[pixel] = MAX_OFFSTREET_WALK_METERS;
+                    distances1_mm[pixel] = MAX_OFFSTREET_WALK_METERS;
+                } else { //point is inside super-grid
+                    int sourcePixel = sourceRow * superGrid.width + sourceColumn;
+                    LOG.info(Integer.toString(sourcePixel));
```

Comment:
This line is causing a lot of log noise, and is probably greatly slowing down the copy operation.

Reply:
Stray log statement, will remove.